src/cpp/shared/combat/army.cpp

```diff
@@ -589,6 +597,69 @@ void army::DoAttack(int isRetaliation) {
 
 }
 
+void army::DoHydraAttack(int isRetaliation) {
```

Comment:
Can you add a comment about what you changed from the original code?

Also, please explain the bug.

Reply:
added a comment
It was just a problem with displaying combat message